climakitae/explore/vulnerability.py

```diff
@@ -107,6 +203,12 @@ def check_and_set_params(
     if approach not in ["time", "warming_level"]:
         raise ValueError(f"Passed in approach must either be 'time' or 'warming_level'")
 
+    # Temp throw error with WL showing future support
+    if approach == "warming_level":
+        raise ValueError(
+            "Warming level integration still under construction! Only time-based approach available for now."
```

Comment:
This validation check should prevent any attempts at using WL approach currently.